src/Orleans/Cancellation/GrainCancellationTokenSource.cs

```diff
@@ -1,14 +1,14 @@
-﻿using System;
-using System.Threading.Tasks;
-using Orleans.CodeGeneration;
+﻿using Orleans.CodeGeneration;
 using Orleans.Serialization;
+using System;
+using System.Threading.Tasks;
```

Comment:
There was no real reason to flip them, right? It's just some tool that did it automatically?